.github/workflows/build_and_test.yml

```diff
@@ -42,9 +42,11 @@ jobs:
             mllib-local, mllib,
             yarn, mesos, kubernetes, hadoop-cloud, spark-ganglia-lgpl
           - >-
-            pyspark-sql, pyspark-mllib, pyspark-resource
+            pyspark-sql, pyspark-resource
           - >-
-            pyspark-core, pyspark-streaming, pyspark-ml
+            pyspark-core, pyspark-streaming
+          - >-
+            pyspark-ml, pyspark-mllib
```

Comment:
@dongjoon-hyun, BTW do you have a GitHub Action workflow example of " The total execution time of two Python GitHub Action jobs is over 3 hours."? I just took a quick look and seems it's usually 1 hour 30 mins. For example, it has a change in core which probably runs most of tests: https://github.com/apache/spark/runs/1236699345

Reply:
@HyukjinKwon . I mean the sum of two jobs.

Reply:
In my very previous `Zstd-jni` update jobs, the sum was 3 hour 14 minutes like the following.
- https://github.com/apache/spark/runs/1240470628 (1 hour 35 min)
- https://github.com/apache/spark/runs/1240470634 (1 hour 39 min)

Reply:
We cannot rebalance between two jobs because of this. We need at least 3 jobs to lowe

Reply:
I updated the PR description more clearly.

Reply:
ahh gotya